packages/node_modules/@webex/plugin-meetings/src/member/util.js

```diff
@@ -126,8 +126,27 @@ MemberUtil.isMuted = (status, accessor) => {
   return null;
 };
 
+/**
+ * utility method for getting the recording member for later comparison
+ * @param {Object} controls
+ * @returns {String}
+ */
+MemberUtil.getRecordingMember = (controls) => {
+  if (!controls) {
+    return null;
+  }
+  if (controls.record && controls.record.recording) {
+    if (controls.record.meta) {
+      return controls.record.meta.modifiedBy;
+    }
+  }
+
+  return null;
+};
+
 /**
  * @param {Object} participant the locus participant
+ * @param {Object} locus the meeting instance
```

Comment:
JSDoc updated, but function wasn't

Reply:
yeah i thought i had pushed this one, the jsdoc is inaccurate